components/Snackbar.html

```diff
@@ -0,0 +1,97 @@
+                <a href="https://github.com/akash-1318"><i class='bx bxl-github'></i></a>
+                <a href="https://www.linkedin.com/in/akash-sharma-0251051a1"><i class='bx bxl-linkedin'></i></a>
+                <a href="https://twitter.com/Akasharma18"><i class='bx bxl-twitter'></i></a>
+            </div>
+        </header>
+        <section class="main__section">
+            <aside class="sidebar">
+                <p class="sidebar__heading">Components</p>
+                <ul class="sidebar__menu">
+                    <li class="sidebar__menu-link"><a href="Avatar.html">
+                            <p>Avatar</p>
+                        </a></li>
+                    <li class="sidebar__menu-link"><a href="Alert.html">
+                            <p>Alert</p>
+                        </a></li>
+                    <li class="sidebar__menu-link"><a href="Badge.html">
+                            <p>Badge</p>
+                        </a></li>
+                    <li class="sidebar__menu-link"><a href="Button.html">
+                            <p>Button</p>
+                        </a></li>
+                    <li class="sidebar__menu-link"><a href="Card.html">
+                            <p>Card</p>
+                        </a></li>
+                    <li class="sidebar__menu-link"><a href="#">
+                            <p>Image</p>
+                        </a></li>
+                    <li class="sidebar__menu-link"><a href="Input.html">
+                            <p>Input</p>
+                        </a></li>
+                </ul>
+            </aside>
+            <div class="main__grid-item">
+                <div class="component">
+                    <p class="component__heading">Snackbar</p>
+                    <p class="component__text">Snackbars are used to show small and quick messages and can contain a single action button like Dismiss or cancel. </p>
+                    <p class="component__heading-primary">Text and dismiss snackbar</p>
+                    <p class="component__sub-text">
+                        Here are simple text and dismiss snackbars you can implement in your 
+                        project. Just add different classes for different snackbar along with 
+                        class <span class="comp__detail-text">.snackbar</span> like for text use <span class="comp__detail-text">.text-snackbar</span>
+                    </p>
+                    <div class="component__view">
+                        <div class="snackbar text-snackbar">
+                            <p class="snackbar-first-para">Can't send photo. Retry in 5 seconds.</p> 
+                        </div>
+            
+                        <div class="snackbar snackbar__accent">
+                            <p class="snackbar-first-para">Can't send photo. Retry in 5 seconds.</p>
+                            <p class="snackbar-second-para">Undo</p>
+                            <i class='bx bx-x'></i>
```

Comment:
instead o using first para and second para you can use snackbar__title and snackbar__subtitle